internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -1191,6 +1227,72 @@ func configMultiZoneWithShards(orgID, projectName, name string, numShardsFirstZo
 	`, orgID, projectName, name, numShardsFirstZone, numShardsSecondZone, selfManagedSharding)
 }
 
+func checkMultiZoneWithShardsOldSchema(name string, numShardsFirstZone, numShardsSecondZone int) resource.TestCheckFunc {
+	return checkAggr(
+		[]string{"project_id"},
+		map[string]string{
+			"name":         name,
+			"disk_size_gb": "60",
+			"replication_specs.0.region_configs.0.electable_specs.0.disk_size_gb": "60",
+			"replication_specs.0.region_configs.0.analytics_specs.0.disk_size_gb": "60",
+			"replication_specs.0.num_shards":                                      strconv.Itoa(numShardsFirstZone),
+			"replication_specs.1.num_shards":                                      strconv.Itoa(numShardsSecondZone),
+		})
+}
+
+func configShardedOldSchemaDiskSizeGBElectableLevel(orgID, projectName, name string) string {
+	return fmt.Sprintf(`
+		resource "mongodbatlas_project" "cluster_project" {
```

Comment:
can't we use ProjectIDExecution?

Reply:
will adjust in follow up PR, I see we have a mix of both for the advanced_cluster tests